### ISSUE TITLE
Write a full description in the ReadME

### ISSUE THREAD

**Issue by DonaldTsang** [zbigniewzolnierowicz/washr#2]
Is it possible to describe what kind of social network this repo is, as there are many social network formats. Is this a Booru, a Chan, a Twitter-like or something else entirely?
Also: ELI5 of the format.

**Comment by zbigniewzolnierowicz** [comment]
Okay, I took your ideas to heart, and updated the README. Thank you for writing the first issue on here 🎉

**Comment by zbigniewzolnierowicz** [comment]
Oh, I also added the link to the web app on Firebase, so you can check it out.

**Comment by DonaldTsang** [comment]
Thanks for the help, if you want a feature list of 4chan-esques I will provide them for reference.
